app/assets/stylesheets/_forms.scss

```diff
@@ -8,7 +8,7 @@ fieldset {
 input,
 label,
 select {
-  display: block;
+  display: inline;
```

Comment:
Why not leave this one as `block`? I very rarely use anything but block on inputs and I think most people like that default